Directory.Build.props

```diff
@@ -36,7 +36,9 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)AzureSDKClient.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
```

Comment:
Required for InternalsVisibleTo

Reply:
That shouldn't be the case, AFAIK. InternalsVisibleTo only requires a key (second parameter) if the project is signed. All of our projects in Azure PowerShell had blocks like this in the AssemblyInfo:
https://github.com/Azure/azure-powershell/blob/6f2df0317fb7f7170a8effea7a897ef09f00259e/src/PrivateDns/PrivateDns/Properties/AssemblyInfo.cs#L49-L53

So, unless we're signing now (which is fine if we are), we should likely have a build that does signing (Release) and a build that doesn't (Debug). But, signing isn't required for InternalsVisibleTo.

Reply:
Yeah, you are right, if both are unsigned InternalsVisibleTo would still work. I don't think there is any benefit to making signing conditional though.